documents/policy/policy.tex

```diff
@@ -165,9 +165,21 @@ \section{Prototypes}
 file that details the objective of the prototype and the outcome. The
 outcome will serve as a "lessons learned" for use in the project.
 
+\section{Tardiness}
+
+Excessive and repeated lateness will be met with the formal issuing of a yellow-card.  Being late will be considered being more than 10 minutes late without any sort of valid excuse.
+
+\section{Pull Requests}
+
+It is expected that people check pull requests at least once a day.  If an item is not reviewed by a team member within 24 hours, any member can add themselves as a reviewer and 'yoink' the review from you. Although no formal action is taken, it reflects extremely badly on you if you're always letting your reviews get yoinked by more attentive teammates
+
+\section{Read and Understood}
+A R&U (read and understood) response is mandatory in slack for messages, if you do not respond in slack with a :ladies\_and\_gentleman:, the PM (project manager) will PM (private message) you to ensure you've read the message and understand the implications
```

Comment:
hahah this is fine but you could probably just say you should at the very least add a slack reaction to denote you read it.